lib/utils.ts

```diff
@@ -111,3 +113,20 @@ export function createSurfaceFromWidget(widget: Gtk.Widget) {
     widget.draw(cr)
     return surface
 }
+
+/*
+// get anchor position from option in user config.
+// currently accepts either notification or OSD, should these be merged?
+*/
+export function getPosition (pos: NotificationAnchor | OSDAnchor): ("top" | "bottom" | "left" | "right")[] {
+    const positionMap: { [key: string]: ("top" | "bottom" | "left" | "right")[] } = {
+        "top": ["top"],
+        "top right": ["top", "right"],
+        "top left": ["top", "left"],
+        "bottom": ["bottom"],
+        "bottom right": ["bottom", "right"],
+        "bottom left": ["bottom", "left"]
```

Comment:
```suggestion
        "bottom left": ["bottom", "left"],
        "right": ["right"],
        "left": ["left"],
```

Left and Right anchors seem to be missing. Let's add this here.